List disputed names

Lukewh opened the pull request:
Fixes https://github.com/CanonicalLtd/snap-squad/issues/921

# USE STAGING API'S IF YOU'RE GOING TO CREATE SOME DISPUTES

Implemented as per:
![screenshot 2019-03-06 at 11 04 01 am](https://user-images.githubusercontent.com/479384/53900289-1e292200-4034-11e9-996d-a0c9dad09a7e.png)

# USE STAGING API'S IF YOU'RE GOING TO CREATE SOME DISPUTES

## QA
- Pull the branch
# USE STAGING API'S IF YOU'RE GOING TO CREATE SOME DISPUTES
- `./run`
- Visit http://0.0.0.0:8004/snaps
- If you have disputed names, they should be displayed as red with a warning icon
- Other registered snaps (unpublished) should be shown, hovering over the row should show a link to documentation

# USE STAGING API'S IF YOU'RE GOING TO CREATE SOME DISPUTES

## Screenshot
![screenshot_2019-03-06 my published snaps linux software in the snap store 1](https://user-images.githubusercontent.com/479384/53899946-482e1480-4033-11e9-9f0b-49662d2a7254.png)

# USE STAGING API'S IF YOU'RE GOING TO CREATE SOME DISPUTES

hughesjordan commented:
Juan mentioned a few small tweaks yesterday.. to drop the red an have 'Snap name disputed' in brackets:
![screenshot 2019-03-07 at 9 58 51 am](https://user-images.githubusercontent.com/17141586/53948693-4e1e0700-40c0-11e9-9317-ab3853837ccf.png)

Also probably a separate issue but worth noting, we will need to display an icon in the 'My published snaps' table too:
![screenshot 2019-03-07 at 10 07 46 am](https://user-images.githubusercontent.com/17141586/53948959-ef0cc200-40c0-11e9-998e-c9974a1f5999.png)

Lukewh commented:
Updated screenshot:
![screenshot_2019-03-07 my published snaps linux software in the snap store](https://user-images.githubusercontent.com/479384/53957717-fb9c1500-40d6-11e9-8c77-67be7273ee6c.png)

I also added the tooltip to the main table:
![screenshot_2019-03-07 my published snaps linux software in the snap store 1](https://user-images.githubusercontent.com/479384/53957840-559cda80-40d7-11e9-8857-02a0ef687721.png)

therealjuan commented:
Just one small comment, can we have the warning on the same position on both tables? After the snap name?

Lukewh commented:
![Screenshot_2019-03-11  My published snaps — Linux software in the Snap Store ](https://user-images.githubusercontent.com/479384/54119126-89844280-43ec-11e9-9af4-318632768b3c.png)
Moved @therealjuan

jpmartinspt commented:
@Lukewh Code looks fine, if anything I would suggest http://jinja.pocoo.org/docs/2.10/templates/#with-statement, just to avoid writing that comparison all the time. Maybe not for now, but might come in handy if we keep adding status checks to that table.

jpmartinspt commented on templates/publisher/account-snaps.html:
```diff
@@ -169,18 +179,32 @@ <h3 class="p-heading--five">Join the community</h3>
+                  </td>
+                  <td>
+                    {% if dispute_pending %}
+                      <span class="p-snapcraft-dispute-list__muted">(Snap name disputed)</span>
```
I think (snap name disputed) makes it sound like the dispute is not pending anymore. I would maybe use (snap name in dispute). Thoughts @therealjuan @Lukewh?

therealjuan replied:
Name dispute in progress?

Lukewh replied:
Updated :+1:

jpmartinspt replied:
:+1:
